pages/inventory.js

```diff
@@ -150,6 +150,12 @@ export default function Inventory() {
     ref.once("value")
     .then(function(resp) {
       let res = resp.val();
+      // Insert new col "Case Count"
+      for (const key in res) {
+        res[key].caseCount = ((parseInt(res[key].packSize) !== 1) && (parseInt(res[key].count) > 1))
+          ? parseInt(res[key].count) / parseInt(res[key].packSize)
```

Comment:
I think we can keep the case count being a whole number and just have it be floor(parseInt(res[key].count) / parseInt(res[key].packSize)) since we primarily keep track of inventory by whole cases and don't really care about partially used cases

Reply:
Consulted with Natalia and Sarah, who say that a decimal is okay.